lightning/src/ln/router.rs

```diff
@@ -346,6 +346,58 @@ pub struct RouteHint {
 	pub htlc_minimum_msat: u64,
 }
 
+/// Details of a channel, as returned by Router::list_graph
```

Comment:
Why add new structs? We already store them in the message format, why not just return those?

Reply:
Right maded themp public, but added their identifier for each